frontend/src/fakeNames.js

```diff
@@ -13,7 +13,7 @@ class ShowFakeNames extends Component {
           return (
             <div>
               <ul>
-                <li className="fakeName">{element}</li>
+                <li className={styles.names}>{element}</li>
```

Comment:
Nice job on having different css files and importing them. Improves the structure